src/meson.build

```diff
@@ -0,0 +1,67 @@
+sources = files(
+  'buildable.vala',
+  'builder.vala',
+  'error.vala',
+  'libyaml-glib.vala',
+  'loader.vala',
+  'writer.vala',
+)
+
+deps = [
+  gobject_introspection_dep,
+  yaml_dep,
+  gmodule_dep,
+]
+
+revision = 0
+current = 1
+age = 0
+current_minus_age = current - age
+project_libversion = '@0@.@1@.@2@'.format(current_minus_age, age, revision)
+
+libglib_yaml = shared_library(
```

Comment:
Use `library` here instead of `shared_library` to enable static builds.

Reply:
ok